options/posix/include/mlibc/posix-sysdeps.hpp

```diff
@@ -89,7 +89,8 @@ int sys_close(int fd);
 [[gnu::weak]] void sys_yield();
 [[gnu::weak]] int sys_sleep(time_t *secs, long *nanos);
 [[gnu::weak]] int sys_fork(pid_t *child);
-[[gnu::weak]] int sys_clone(void *entry, void *user_arg, void *tcb, pid_t *pid_out);
+[[gnu::weak]] int sys_clone(void *tcb, pid_t *pid_out, void *stack);
+[[gnu::weak]] int sys_prepare_stack(void **stack, void *entry, void *user_arg, size_t stack_size, size_t guard_size);
```

Comment:
This definition does not match the definitions in the sysdeps, where an extra `void *tcb` can be found as 4th argument. Only x86_64 linux has a matching definition at this moment.

Reply:
Ah, you're right. That's an easy fix tho.